Gemfile

```diff
@@ -16,5 +16,5 @@ gem "webrick"
 group :jekyll_plugins do
     gem "jekyll-include-cache"
     gem 'jekyll-algolia', "~> 1.0", path: "./jekyll-algolia-dev"
-    gem 'jekyll-remote-include', "~> 1.1.4", github: 'ianjevans/jekyll-remote-include'
+    gem 'jekyll-remote-include', "~> 1.1.4", github: 'nickvigilante/jekyll-remote-include'
```

Comment:
```suggestion
    gem 'jekyll-remote-include', "~> 1.1.6", github: 'nickvigilante/jekyll-remote-include'
```

Reply:
@ianjevans I took the fix from the Slack thread and just bumped it from 1.1.5 to 1.1.6. Check the Gemfile once more, but I think this should be good.